modules/engine/src/shader-inputs.ts

```diff
@@ -89,9 +89,10 @@ export class ShaderInputs<
       const moduleProps = props[moduleName];
       const module = this.modules[moduleName];
 
-      const uniforms = module.getUniforms?.(moduleProps, this.moduleUniforms[moduleName]);
+      const oldUniforms = this.moduleUniforms[moduleName] || {};  
```

Comment:
nit, optional: `this.moduleUniforms[moduleName]` should be non-nullable based on the constructor and types. So either the fallback to `{}` should not be needed, or the types are incorrectly strict?